docs/src/IceNucleationBox.md

```diff
@@ -8,11 +8,6 @@ It is based on [KorolevMazin2003](@cite), as well as the cirrus box model
 The model solves for saturation in a 0-dimensional
   adiabatic parcel raising with constant velocity.
 
-!!! note
-    For now, the equations are written assuming
```

Comment:
Could we also separate this one from the ABIFM PR?

Reply:
Also, would be great to add a parcel example where we just have liquid water grow via condensation using the non-equilibrium microphysics. And show that we can reproduce the typical supersaturation(height) behavior